clients/client-python/gravitino/namespace.py

```diff
@@ -34,13 +33,13 @@ def __init__(self, levels: List[str]):
         self._levels = levels
 
     def to_json(self):
-        return json.dumps(self._levels)
+        return self._levels
```

Comment:
Why do we do this changes?

Reply:
It is a bug in the previous code, it treat a json list as a string, which will lead to failure in deserialization.